main.py

```diff
@@ -1648,5 +1719,5 @@ def test_f2_631g_1p5p5det(self):
         PROFILING = False
     else:
         PROFILING = True
-    doctest.testmod(optionflags=doctest.NORMALIZE_WHITESPACE, raise_on_error=True)
+    doctest.testmod(optionflags=doctest.NORMALIZE_WHITESPACE, raise_on_error=False)
```

Comment:
Why did you change it to False?

Reply:
I changed it while I was testing and forgot to change it back.
I think this:
```
$ python main.py Test_Index 
**********************************************************************
File "/home/kgasperich/work/qe-clean/main.py", line ?, in __main__.compound_idx2
Failed example:
    compound_idx2(2,1)
Expected:
    5
Got:
    4
**********************************************************************
1 items had failures:
   1 of   6 in __main__.compound_idx2
***Test Failed*** 1 failures.
__main__.Test_Index.test_canonical_idx4 ... ok (0.516s)
__main__.Test_Index.test_compound_idx4_reverse_is_canonical ... ok (0.113s)
__main__.Test_Index.test_idx2_reverse ... ok (0.051s)
__main__.Test_Index.test_idx4_reverse ... ok (0.114s)
__main__.Test_Index.test_idx4_reverse_all ... ok (0.275s)
----------------------------------------------------------------------
Ran 5 tests in 1.069s

OK
```

is more useful than this:

```
$ python main.py Test_Index 
Traceback (most recent call last):
  File "/home/kgasperich/work/qe-clean/main.py", line 1722, in <module>
    doctest.testmod(optionflags=doctest.NORMALIZE_WHITESPACE, raise_on_error=True)
  File "/home/kgasperich/miniconda3/lib/python3.9/doctest.py", line 1956, in testmod
    runner.run(test)
  File "/home/kgasperich/miniconda3/lib/python3.9/doctest.py", line 1844, in run
    r = DocTestRunner.run(self, test, compileflags, out, False)
  File "/home/kgasperich/miniconda3/lib/python3.9/doctest.py", line 1483, in run
    return self.__run(test, compileflags, out)
  File "/home/kgasperich/miniconda3/lib/python3.9/doctest.py", line 1384, in __run
    self.report_failure(out, test, example, got)
  File "/home/kgasperich/miniconda3/lib/python3.9/doctest.py", line 1853, in report_failure
    raise DocTestFailure(test, example, got)
doctest.DocTestFailure: <DocTest __main__.compound_idx2 from /home/kgasperich/work/qe-clean/main.py:None (6 examples)>
```

Reply:
No, because in the first example, the test will pass. We will have a green light,
where in the second the tests will fail

Reply:
I found one way to do it; see 3164155